go.mod

```diff
@@ -12,6 +12,8 @@ replace github.com/filecoin-project/boostd-data => ./extern/boostd-data
 
 // replace github.com/filecoin-project/boost-gfm => ../boost-gfm
 
+replace github.com/ipld/frisbii => ../../ipld/frisbii
```

Comment:
This would go away by the time we get to the root PR against `main`?

Reply:
yeah, already gone from here too, using frisbii@v0.4.0 which has what we need